docs/snippets/proc-subscribe-function.md

```diff
@@ -0,0 +1,44 @@
+<!-- Snippet used in the following topics:
+- /docs/functions/subscribing-functions.md
+-->
+The `subscribe` command will connect the function to a set of events, matching a series of filters for Cloud Event metadata
+and a Knative Broker as the source of events, from where they are consumed.
+
+=== "func"
+
+    To subscribe the function to events for a given broker, run the following command:
+
+    ```bash
+    func subscribe --filter type=com.example --filter extension=my-extension-value --source my-broker
+    ```
+
+    To subscribe the function to events for the default broker, run the following command:
+
+    ```bash
+    func subscribe --filter type=com.example --filter extension=my-extension-value
+    ```
+
+=== "kn func"
+
+    To subscribe the function to events for a given broker, run the following command:
+
+    ```bash
+    kn func subscribe --filter type=com.example --filter extension=my-extension-value --source my-broker
+    ```
+
+    To subscribe the function to events for the default broker, run the following command:
+
+    ```bash
+    kn func subscribe --filter type=com.example --filter extension=my-extension-value
+    ```
+
+!!! note
+    When invoking `func deploy` the CLI will create Knative Triggers for the function.
```

Comment:
I'd move this note above the code blocks, line no. 7 of the `.md` file. Otherwise looks OK to me.

Reply:
I've separated that now, and added a more dedicated section, with a smalll focus on the deployment.

Moving the note to far up seems a bit odd to me...